### DISCUSSION TITLE
Is the web browser and the custom cache going to cache automatically?

### DISCUSSION THREAD

**Discussion by sam-93** [arthurfiorette/axios-cache-interceptor#909] [Q&A] [answered]
Suppose you do a web browser request, and receive everything with a cache-control-header.
You also have a custom caching implementation. What would happen? 
Is the web browser and the custom cache going to cache automatically? (double caching, one from browser and another from the custom)
If so, how can I override this behavior, such that only the custom cache takes precedence?

**Comment by arthurfiorette** [comment, marked as answer]
https://axios-cache-interceptor.js.org/config/request-specifics#cache-cachetakeover
